news/4288.feature.rst

```diff
@@ -0,0 +1,2 @@
+On Windows applications built in Windowed mode have their Debug messages
```

Comment:
I suggest

(Windows): Debug messages of windowed applications are send to any attached debugger
instead of showing a message box for each message.